### ISSUE TITLE
Feature: disable borrow UI if appropriate global setting set

### ISSUE THREAD

**Issue by nimaai** [leihs/leihs#1155]
Analog to old borrow UI. Relevant setting: `settings.disable_borrow_section` and `settings.disable_borrow_section_message`.

**Comment by nadjaweisskopf** [comment]
should be integrated with error handling - let's develop the quickest solution.
@nadjaweisskopf check with stakeholders if setting in Admin is still needed.

**Comment by nadjaweisskopf** [comment]
sent a mail to Roger

**Comment by nadjaweisskopf** [comment]
After the input from Roger I would says we do not rebuild the feature. We haven't used it for too long. I don't think it's worth the effort.

**Comment by nimaai** [comment]
@eins78 I think we should perhaps also remove the twin feature: disable manage section.

**Comment by eins78** [comment]
@nimaai I agree. And in general: Both removal warrant the question if this is a breaking change or not, beaucse then we should bump the major to 7. If so, then we might not want to merge it to master right away.

**Comment by nadjaweisskopf** [comment]
@nimaai we'll remove the feature disable manage section as well.

**Comment by nimaai** [comment]
https://github.com/leihs/leihs/commit/a356d15cefef156421cfc7f89f6c84531c6b1784

**Comment by nadjaweisskopf** [comment]
@nimaai where can I test this ticket? On next-environment the features is still shown in the GUI of the ADMIN-section.
